sdorfehs.1

```diff
@@ -343,6 +343,16 @@ Spawn a shell executing
 .Ar shell\-command ,
 showing _NET_WM_PID supporting programs in the given frame instead of
 the frame selected when this program is run.
+.It Ic execw Ar shell\-command
+Spawn a shell executing
+.Ar shell\-command ,
+but wait for it to finish.  This should behave the same as
+.Ar exec
+but will wait the command to finish.  Use timeout(1) if it might hang, or
+sdorfehs will hang itself waiting for completion.  Use the 'exec' shell
```

Comment:
Please use the `.Nm` macro instead of hard-coding `sdorfehs`.

Reply:
I had searched some other man pages for the right macro to use but most man pages seem to use a different set of macros, mostly two upper case letters, which were clearly not the right macro set to intermix so I couldn't find right one, thanks.